lang/cs/Org.Apache.REEF.Network/Group/Config/GroupCommConfigurationOptions.cs

```diff
@@ -61,7 +61,7 @@ internal sealed class SleepTimeWaitingForRegistration : Name<int>
         /// We want it to return as soon as all nodes in the group are registered, So increasing retry count is better than increasing sleep time.
         /// Current default sleep time is 500ms. Default retry is 500. Total is 250000ms, that is 250s, little bit more than 4 min
         /// </remarks>
-        [NamedParameter("Retry times to wait for nodes to be registered", defaultValue: "500")]
+        [NamedParameter("Retry times to wait for nodes to be registered", defaultValue: "100")]
```

Comment:
This change is not related to adding number of forced failure. I don't think we should change it as a side effect of another change, which we're trying to do from time to time. If we decide to change it, it needs to be documented as a separate JIRA